lib/screen/github_doc.dart

```diff
@@ -60,6 +61,7 @@ class _GithubDocPageState extends State<GithubDocPage> {
                 SliverToBoxAdapter(
                     child: Text(
                   title,
+                  key: const Key("github title"),
```

Comment:
please use only 1 name convention for this name. Should be unified with @longbmk. (Should be `VarName` or `varName`, shouldn't contain space.